services/common/src/constants/permissions.ts

```diff
@@ -0,0 +1,26 @@
+export const VIEW_ALL = "role_view";
```

Comment:
I don't love bringing over this file which is kind of redundant, instead of using the USER_ROLES from environment.ts. The Authorization wrapper could be tweaked to use the userHasRole selector instead, which can take values from either.

Reply:
Removed this permission.ts and used environment.ts